test/functional/CommandCallFunctional.js

```diff
@@ -16,20 +16,22 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
 /* eslint-env mocha */
+/* eslint-disable prefer-arrow-callback */
+/* eslint-disable func-names */
```

Comment:
One would think that setting eslint-env mocha would set these sorts of preferences, since Mocha recommends this usage.

Reply:
IKR seems like may want to use https://www.npmjs.com/package/eslint-plugin-mocha like you've suggested before.